packages/ripple/src/runtime/internal/client/render.js

```diff
@@ -128,36 +128,65 @@ export function apply_styles(element, newStyles) {
  * @returns {void}
  */
 export function set_attributes(element, attributes) {
-  for (const key in attributes) {
-    if (key === 'children') continue;
-
-    let value = attributes[key];
-
-    if (is_tracked_object(value)) {
-      value = get(value);
-    }
+	const regularKeys = Object.keys(attributes);
```

Comment:
We want to avoid these expensive calls during the normal fast path where we're not dealing with dynamic elements